fail2ban/server/utils.py

```diff
@@ -146,10 +164,18 @@ def executeCmd(realCmd, timeout=60, shell=True, output=False, tout_kill_tree=Tru
 		"""
 		stdout = stderr = None
 		retcode = None
-		popen = None
+		popen = env = None
+		if varsDict:
+			if shell:
+				# build map as array of vars and command line array:
+				realCmd = Utils.buildShellCmd(realCmd, varsDict)
+			else: # pragma: no cover - currently unused
+				env = _merge_dicts(os.environ, varsDict)
+		realCmdId = id(realCmd)
+		outCmd = lambda level: logSys.log(level, "%x -- exec: %s", realCmdId, realCmd)
```

Comment:
please choose a bit more "proper" name -- e.g. "logCmd".  I was reading code below and got myself asking "what does it output? where to?